### PR TITLE
add h60 PCB

### PR DESCRIPTION
**Pull request by hineybush**
## Description

Add h60 PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hineybush/h60/info.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "h60",
+    "url": "",
+    "maintainer": "hineybush",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT": {
```
Comment (fauxpark) [inline keyboards/hineybush/h60/info.json:8]: This needs updating.

File: keyboards/hineybush/h60/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,5 @@
+//Copyright 2020 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/default/config.h:1]: This file can be deleted.

File: keyboards/hineybush/h60/keymaps/kei/config.h [hunk 3]
```diff
@@ -0,0 +1,5 @@
+//Copyright 2020 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/kei/config.h:1]: As above.

File: keyboards/hineybush/h60/keymaps/via/config.h [hunk 4]
```diff
@@ -0,0 +1,5 @@
+//Copyright 2020 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/via/config.h:1]: As above.

File: keyboards/hineybush/h60/rules.mk [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
```
Comment (fauxpark) [inline keyboards/hineybush/h60/rules.mk:4]:
```suggestion
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

File: keyboards/hineybush/h60/rules.mk [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+SLEEP_LED_ENABLE = yes       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/hineybush/h60/rules.mk:25]:
```suggestion
```

File: keyboards/hineybush/h60/rules.mk [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+VIA_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/hineybush/h60/rules.mk:26]: This needs to go in the `via` keymap's rules.mk, not here.

File: keyboards/hineybush/h60/info.json (hunk 1 above)
Comment (hineybush) [reply keyboards/hineybush/h60/info.json]: I'll just remove this file as VIA does the job better

File: keyboards/hineybush/h60/rules.mk (hunk 6 above)
Comment (hineybush) [reply keyboards/hineybush/h60/rules.mk]: why remove this one line when the new_project script generates it?

File: keyboards/hineybush/h60/rules.mk (hunk 5 above)
Comment (hineybush) [reply keyboards/hineybush/h60/rules.mk:4]: why do these need to be in the code, when I've been told to remove them before? https://github.com/qmk/qmk_firmware/pull/7668#discussion_r359596680

File: keyboards/hineybush/h60/info.json (hunk 1 above)
Comment (zvecr) [reply keyboards/hineybush/h60/info.json]: Except we require it for configurator and *really* should be added as part of this PR.

Its not the issue that via does end user configuration better, but that its part of adding keyboard support to QMK.

File: keyboards/hineybush/h60/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+// Defines the keycodes used by our macros in process_record_user
+
+
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/default/keymap.c:14]:
```suggestion
```

File: keyboards/hineybush/h60/keymaps/kei/keymap.c [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+// Defines the keycodes used by our macros in process_record_user
+
+
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/kei/keymap.c:14]:
```suggestion
```

File: keyboards/hineybush/h60/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+enum layer_names {
+    _BASE,
+    _FN1,
+    _FN2,
+    _FN3
+};
+
+// Defines the keycodes used by our macros in process_record_user
+
+
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/via/keymap.c:16]:
```suggestion
```

File: keyboards/hineybush/h60/h60.h [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+          k401, k402,                   k406,                   k410, k411                    \
+) { \
+    { k000, k001, k002, k003, k004, k005, k006, k007, k008, k009, k010, k011, k012, k013 }, \
+    { k100, k101, k102, k103, k104, k105, k106, k107, k108, k109, k110, k111, k112, k113 }, \
+    { k200, k201, k202, k203, k204, k205, k206, k207, k208, k209, k210, k211, k212, k213 }, \
+    { k300, k301, k302, k303, k304, k305, k306, k307, k308, k309, k310, k311, k312, KC_NO}, \
+    { KC_NO,k401, k402, KC_NO,KC_NO,KC_NO,k406, KC_NO,KC_NO,KC_NO,k410, k411, KC_NO,KC_NO} \
+}
+
+#define LAYOUT_tsangan_hhkb(\
```
Comment (noroadsleft) [inline keyboards/hineybush/h60/h60.h:85]:
```suggestion
#define LAYOUT_60_tsangan_hhkb(\
```

File: keyboards/hineybush/h60/h60.h [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+    k400, k401, k402,                   k406,                   k410, k411, k412              \
+) { \
+    { k000, k001, k002, k003, k004, k005, k006, k007, k008, k009, k010, k011, k012, k013 }, \
+    { k100, k101, k102, k103, k104, k105, k106, k107, k108, k109, k110, k111, k112, k113 }, \
+    { k200, k201, k202, k203, k204, k205, k206, k207, k208, k209, k210, k211, k212, k213 }, \
+    { k300, k301, k302, k303, k304, k305, k306, k307, k308, k309, k310, k311, k312, KC_NO}, \
+    { k400, k401, k402, KC_NO,KC_NO,KC_NO,k406, KC_NO,KC_NO,KC_NO,k410, k411, k412, KC_NO} \
+}
+
+#define LAYOUT_60_tsangan(\
```
Comment (noroadsleft) [inline keyboards/hineybush/h60/h60.h:99]:
```suggestion
#define LAYOUT_60_tsangan_split_rshift(\
```

File: keyboards/hineybush/h60/rules.mk [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = yes       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+
```
Comment (noroadsleft) [inline keyboards/hineybush/h60/rules.mk:25]:
```suggestion
LAYOUTS = 60_ansi 60_ansi_split_bs_rshift 60_hhkb 60_tsangan_hhkb
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**